db/migrate/20151112110911_add_event_id_to_event_logs.rb

```diff
@@ -0,0 +1,6 @@
+class AddEventIdToEventLogs < ActiveRecord::Migration
+  def change
+    add_column :event_logs, :event_id, :integer
+    change_column :event_logs, :event, :string, default: ''
```

Comment:
Indeed! Against my VM database, with ~500k rows it took a good few minutes to complete.